deployment/stacks/imports_included/parent.ts

```diff
@@ -27,5 +34,11 @@ export class ImportsIncludedParentStack extends Stack {
       scope: this,
       stage,
     });
+    createUploadsComponents({
+      scope: this,
+      stage,
+      loggingBucket,
+      isDev: false,
```

Comment:
```suggestion
      isDev,
```
Line 19 hardcodes the value so let's just do it once.